README.md

```diff
@@ -75,17 +75,18 @@ This repository was created using `examples/runDecamTest.sh` from the `validate_
 To fully recreate this Butler `repo` from the `raw` data, set the `mapper` and add the `ingesetImages.py` step:
 
 ```
-setup validation_data_decam
+setup validation_data_decam -t v14_0
```

Comment:
This doesn't actually do anything because the ups table file is empty.

Please either fix the ups table file or these instructions so that the correct software is setup

Reply:
Thanks.  Done.